pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
 
-`sudo deborphan | xargs sudo apt-get remove`
+`ls M* | xargs rm`
+
+- creating variables: copy all files that start with 'M' to the Desktop
```

Comment:
Let's choose some variant and move on. Honestly i tend to 2nd variant, but I understand that it is not complete and not enough for understanding.